runtime/manta/src/lib.rs

```diff
@@ -1385,33 +1404,33 @@ impl_runtime_apis! {
                 type AccountIdConverter = LocationToAccountId;
 
                 fn valid_destination() -> Result<MultiLocation, BenchmarkError> {
-                 Ok(DotLocation::get())
+                    Ok(DotLocation::get())
                 }
 
-                fn worst_case_holding() -> MultiAssets {
+                fn worst_case_holding(depositable_count: u32) -> MultiAssets {
                     // A mix of fungible, non-fungible, and concrete assets.
-                    const HOLDING_FUNGIBLES: u32 = 100;
-                    const HOLDING_NON_FUNGIBLES: u32 = 100;
+                    let holding_non_fungibles = crate::xcm_config::MaxAssetsIntoHolding::get() / 2 - depositable_count;
+                    let holding_fungibles = holding_non_fungibles - 1;
                     let fungibles_amount: u128 = 100;
-                    let mut assets = (0..HOLDING_FUNGIBLES)
+                    let mut assets = (0..holding_fungibles)
                         .map(|i| {
                             MultiAsset {
                                 id: Concrete(GeneralIndex(i as u128).into()),
                                 fun: Fungible(fungibles_amount * i as u128),
                             }
                         })
                         .chain(core::iter::once(MultiAsset { id: Concrete(Here.into()), fun: Fungible(u128::MAX) }))
-                        .chain((0..HOLDING_NON_FUNGIBLES).map(|i| MultiAsset {
+                        .chain((0..holding_non_fungibles).map(|i| MultiAsset {
                             id: Concrete(GeneralIndex(i as u128).into()),
                             fun: NonFungible(asset_instance_from(i)),
                         }))
                         .collect::<Vec<_>>();
 
-                        assets.push(MultiAsset{
-                            id: Concrete(MantaLocation::get()),
-                            fun: Fungible(1_000_000 * MANTA),
-                        });
-                        assets.into()
+                    assets.push(MultiAsset {
+                        id: Concrete(DotLocation::get()),
```

Comment:
why DotLocation here ?

Reply:
some of `pallet_xcm_benchmarks` cases can transfer dot to relaychain